drivers/src/memory_layout.rs

```diff
@@ -29,27 +32,28 @@ pub const ROM_DATA_ORG: u32 = 0x50000000;
 pub const CFI_STATE_ORG: u32 = 0x500003E4; // size = 6 words
 pub const BOOT_STATUS_ORG: u32 = 0x500003FC;
 pub const MAN1_ORG: u32 = 0x50000400;
-pub const MAN2_ORG: u32 = 0x50001C00;
-pub const FHT_ORG: u32 = 0x50003400;
-pub const LDEVID_TBS_ORG: u32 = 0x50003C00;
-pub const FMCALIAS_TBS_ORG: u32 = 0x50004000;
-pub const RTALIAS_TBS_ORG: u32 = 0x50004400;
-pub const PCR_LOG_ORG: u32 = 0x50004800;
-pub const MEASUREMENT_LOG_ORG: u32 = 0x50004C00;
-pub const FUSE_LOG_ORG: u32 = 0x50005000;
-pub const DPE_ORG: u32 = 0x50005400;
-pub const PCR_RESET_COUNTER_ORG: u32 = 0x50006800;
-pub const AUTH_MAN_IMAGE_METADATA_LIST_ORG: u32 = 0x50006C00;
-pub const DATA_ORG: u32 = 0x50007000;
-
-pub const STACK_ORG: u32 = 0x5001A000;
-pub const ROM_STACK_ORG: u32 = 0x5001C000;
-
-pub const ESTACK_ORG: u32 = 0x5001F800;
-pub const ROM_ESTACK_ORG: u32 = 0x5001F800;
-
-pub const NSTACK_ORG: u32 = 0x5001FC00;
-pub const ROM_NSTACK_ORG: u32 = 0x5001FC00;
+pub const MAN2_ORG: u32 = MAN1_ORG + MAN1_SIZE;
```

Comment:
Every time there is a change to one of the structs, we have to manually adjust the addresses. This way, it's more robust. I don't see a point in having tests that are checking against a hard-coded numeric value.

Reply:
The reason the tests were comparing to hard-coded values was to confirm that a previous refactoring didn't change the values and break binary compatibility. Since 2.x is moving everything around anyways, I think this is ok. In fact, we can probably get rid of all these constants and just write a goldendata test to confirm that things don't move around unexpectedly during future refactorings.

Reply:
We'll address this as a separate change.